src/components/ItemStatus/ItemStatus.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Popup } from 'decentraland-ui'
+
+import { SyncStatus } from 'modules/item/types'
+import { Props } from './ItemStatus.types'
+import './ItemStatus.css'
+
+export default class ItemStatus extends React.PureComponent<Props> {
```

Comment:
ItemStatus and CollectionStatus share the same css and code. They could be abstracted to a general Status component on which a `type: 'item' | 'collection'` prop is provided.
The container has a minimal difference in both, but it can be handled by using this prop from ownProps